platform-dom/src/main/java/com/softicar/platform/dom/elements/number/decimal/DomBigDecimalInput.java

```diff
@@ -4,20 +4,17 @@
 import com.softicar.platform.dom.elements.input.AbstractDomNumberInput;
 import com.softicar.platform.dom.input.DomTextInput;
 import java.math.BigDecimal;
-import java.math.RoundingMode;
+import java.util.Optional;
 
 /**
  * A {@link DomTextInput} that verifies that the user entered a valid
  * {@link BigDecimal} value.
  *
  * @author Semsudin Mekanic
+ * @author Oliver Richers
  */
 public class DomBigDecimalInput extends AbstractDomNumberInput {
```

Comment:
this input now does not force any scaling on the user

Reply:
![image](https://user-images.githubusercontent.com/83839745/144024162-b0359f29-cb52-4ac9-8056-4fab96f2868b.png)